.ci/scripts/check-beats-version.sh

```diff
@@ -2,4 +2,4 @@
 
 set -xeu
 
-exec grep "BEATS_VERSION?=$BRANCH_VERSION" Makefile
+exec grep "BEATS_VERSION?=$BRANCH_NAME" Makefile
```

Comment:
**_question_**: should we fail early if BRANCH_NAME is empty ?

Reply:
I don't think it'll matter in practice, let's come back and improve that if it needed.